miniapps/nurbs/nurbs_printfunc.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
```

Comment:
This small example could be removed or could become a unit test. I am fine with either option.

Reply:
Good suggestion! I was thinking about something like this as well but I was not able to come up with a good test. The function does not return anything or changes anything that can be verified as correct, it only prints something.

Reply:
Usually a unit test checks that some result matches. In this case, there are many numbers in the output, so the test would need to check the norm of the difference between the output numbers and some hard-coded numbers, as vectors. I suppose for this test it is not really worth the effort. It may be more of a test for knot vectors than for the printing, but other tests should fail if knot vectors stop working properly. So it is probably best just to remove this example before merging.